src/formio/templates/radio.ejs

```diff
@@ -13,6 +13,7 @@
     ref="input"
     value="{{item.value}}"
     id="{{ctx.id}}{{ctx.row}}-{{item.value}}"
+    aria-describedby="{{ctx.id}}{{ctx.row}}-{{item.value}}-description"
```

Comment:
if there's no item description, there's an aria-describedby pointer to an ID that doesn't exist. That doesn't seem valid - probably best to only include the attribute if there is a description:

```suggestion
    {% if (item.description) %}aria-describedby="{{ctx.id}}{{ctx.row}}-{{item.value}}-description"{% } %}
```